frontend/src/TooljetDatabase/Table/index.jsx

```diff
@@ -89,6 +89,80 @@ const Table = ({ collapseSidebar }) => {
   const [isCellUpdateInProgress, setIsCellUpdateInProgress] = useState(false);
   const [referencedColumnDetails, setReferencedColumnDetails] = useState([]);
 
+  const [cachedOptions, setCahedOptions] = useState([]);
```

Comment:
Since cached options are **Object** by default `const dataToCache = {};` as mentioned in line no 140. Can we make cachedOptions default state as empty object.